scripts/zones/Northern_San_dOria/IDs.lua

```diff
@@ -31,7 +31,7 @@ zones[xi.zone.NORTHERN_SAN_DORIA] =
         GP_OBTAINED                   = 7017,  -- Obtained: <number> guild points.
         NOT_HAVE_ENOUGH_GP            = 7018,  -- You do not have enough guild points.
         RENOUNCE_CRAFTSMAN            = 7034,  -- You have successfully renounced your status as a [craftsman/artisan/adept] of the [Carpenters'/Blacksmiths'/Goldsmiths'/Weavers'/Tanners'/Boneworkers'/Alchemists'/Culinarians'] Guild.
-        CONQUEST_BASE                 = 7288,  -- Tallying conquest results...
+        CONQUEST_BASE                 = 7274,  -- Tallying conquest results...
```

Comment:
The Ids should not be changed. That new ID stands for
`It's all the fault of those Royal Knights! Does Prince Trion think to undo all that my son has wrought? Does he?`

Same applies for other IDs in other files, with different messages.

Reply:
Ah.. this made me waste so much time. I think it was an issue with my client? Otherwise I can't explain. My offsets were different. I recently switched to retail for caps so that may be it :S

Reply:
Changed back